src/gui/guiFormSpecMenu.cpp

```diff
@@ -4658,6 +4653,12 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 						}
 
 						if (amount > 0) {
+							if (m_left_dragging) {
```

Comment:
If `!m_left_dragging`, we already abort earlier:

https://github.com/minetest/minetest/blob/2a40c2af39e917d65244e7865e34f068d9237558/src/gui/guiFormSpecMenu.cpp#L4621-L4626

so `m_left_dragging` is always true here, isn't it?